src/storage/StorageServer.cpp

```diff
@@ -168,6 +170,28 @@ bool StorageServer::start() {
   options.dataPaths_ = dataPaths_;
 
   metaClient_ = std::make_unique<meta::MetaClient>(ioThreadPool_, metaAddrs_, options);
+
+#ifdef BUILD_STANDALONE
+  if (FLAGS_add_local_host) {
```

Comment:
1. Does meta always started before storage in standalone?
2. Do we need to move `addHosts` after `waitForMetadReady`?

Reply:
1. yes, standalone daemon will wait meta service start then start storage.
2. we can't if we don't addhosts first,  wait will stuck there.

Reply:
👌